### PR TITLE
Document scenario options

### PR DESCRIPTION
**Pull request by ekohl**
This takes the options from the example scenario and documents them. It also does some grouping of options.

If done right, all code throughout Kafo should use the constants rather than symbols. That way it's easy to see where a scenario option is actuall used throughout the code.

### CODE REVIEW

**Comment by ekohl** [comment]
For those unfamiliar with yard: a preview can be rendered using `bundle exec yard && xdg-open doc/Kafo/ScenarioOptions.html`

**Comment by ehelms** [comment]
For me this has benefits and can go in as the starting place to begin using the constants across the code base. If you could rebase @ekohl I'd like to merge this.

File: lib/kafo/scenario_options.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+module Kafo
+  # A class containing constants for all scenario options
+  class ScenarioOptions
```
Comment (ekohl) [inline lib/kafo/scenario_options.rb:3]: Before I rebase: should this be singular or plural? I think `Kafo::ScenarioOption::NAME` reads better than `Kafo::ScenarioOptions::Name`. Thoughts?

Comment (ehelms) [reply lib/kafo/scenario_options.rb:3]: Singular.

**Comment by ehelms** [comment]
Could you rebase and address the singular comment above?

**Comment by ekohl** [comment]
Rebased

**Review by ehelms — APPROVED** [review]

**Pull request merged**